mlflow/pyfunc/backend.py

```diff
@@ -137,9 +195,14 @@ def copy_model_into_container(dockerfile_context_dir):
         )
 
 
-def _execute_in_conda_env(conda_env_path, command, install_mlflow, command_env=None):
+def _execute_in_conda_env(
+    conda_env_path, command, install_mlflow, command_env=None, synchronous=True, preexec_fn=None
+):
     if command_env is None:
         command_env = os.environ
+
+    command_env['PIP_NO_INPUT'] = '1'
```

Comment:
In databricks,
the driver side we set env variable `PIP_NO_INPUT` to be 1, but in executor side it does not set the env variable.
To keep consistency, I set it here.

Reply:
It force PIP to ignore console input, (otherwise pip might prompt "yes or no" and ask stdin input)

Reply:
Nice! Let's add a small inline comment explaining this